src/main/java/frc/robot/lib/MathThings.java

```diff
@@ -22,23 +22,30 @@ public static double angleDiffDeg(double ang1, double ang2){
        return ang2-ang1;
     }
 
-    public static int[] randomIndexes(int size){
-        int[] out = new int[size];
-        for(int i = 0; i<out.length; i++)
-            out[i] = i;
-
-        for(int i = 0; i<out.length; i++){
-            int rand1 = (int)(Math.random()*out.length);
-            int rand2 = (int)(Math.random()*out.length);
-            int held = out[rand1];
-            out[rand1] = out[rand2];
-            out[rand2] = held;
-        }
-        return out;
-    }
 
-    public static double poseDist(Pose2d pose1, Pose2d pose2){
-        return Math.sqrt(Math.pow(pose1.getX() - pose2.getX(),2)+Math.pow(pose1.getY() - pose2.getY(),2));
+
```

Comment:
This is a general comment: but for all of the methods in this class and in the grabber subsystem and all the stuff we write in the future we should add as many comments explaining the methods as we can. Most of these methods should be self-explanatory in theory, but thinking from the perspective of a new coder trying to look at this and figure out what is going on, it helps to have as much explanation as possible even if it feels like you're explaining it to a five-year-old. This is also something Reilly has been really pushing for which is why I'm emphasizing it. If you want me to go back through the code and make comments for everything you've written then that can be one of my tasks, but I'll probably have to ask y'all for explanations on some of the stuff lol.

Reply:
I think moreso even than this, having accurate method names and seperating large methods into many methods goes a very long way. if you need a comment to know what a method does, it probably isnt named well. to know how its done maybe requires a comment describing certain math formulas.